modules/oadp-backup-restore-cr-issues.adoc

```diff
@@ -48,10 +48,17 @@ $ oc -n {namespace} exec deployment/velero -c velero -- ./velero \
 $ oc delete backup <backup> -n openshift-adp
 ----
 +
-You do not need to clean up the backup location because a `Backup` CR in progress has not uploaded  files to object storage.
+You do not need to clean up the backup location because a `Backup` CR in progress has not uploaded files to object storage.
 
 . Create a new `Backup` CR.
 
+. View the Velero backup details
++
+[source,terminal, subs="+quotes"]
+----
+$ velero backup describe _<backup-name>_ --details
```

Comment:
I don't think we use the italicized variables in OCP docs. I see a few examples, but not many.

Reply:
I think that the discrepancy with the italics isn't sufficient to hold up the merge. Please consider this matter for a follow-up PR.